cli/script/command-executor.ts

```diff
@@ -1347,6 +1365,47 @@ function serializeConnectionInfo(accessKey: string, preserveAccessKeyOnLogout: b
     log(`\r\nSuccessfully logged-in. Your session file was written to ${chalk.cyan(configFilePath)}. You can run the ${chalk.cyan("code-push logout")} command at any time to delete this file and terminate your session.\r\n`);
 }
 
+function sessionList(command: cli.ISessionListCommand): Promise<void> {
+    throwForInvalidOutputFormat(command.format);
+
+    return sdk.getAccessKeys()
```

Comment:
I'm not sure. Part of me still feels like whoever consumes the SDK should be made aware that sessions and access keys are really the same thing differentiated only with the `isSession` fieldd. And in some ways, even CLI end users would still be aware of this (when the user logs in, the CLI says "enter your access key"). Also, the `removeAccessKey`, `editAccessKey` SDK functions are all currently designed to work for session keys as well. The fact that we have a special method `removeSessions` is really just a helper method to delete access keys which have `isSession` set to `true` and a `createdBy` set to the specified value. Unless we say we want to create separate SDK functions for all of those, but I'm not sure what the value is there.